### ISSUE TITLE
Callback called multiple times

### ISSUE THREAD

**Issue by cpetta** [cpetta/GulpDev#2]
When .jpg files are in the dev folder, gulp-imagemin fails and gives the following error message. 

```
Error: Callback called multiple times
    at DestroyableTransform.afterTransform (C:\gulpdev\node_modules\gulp-cache\node_modules\readable-stream\lib\_stream_transform.js:82:31)
    at EventEmitter.signals.on.err (C:\gulpdev\node_modules\gulp-cache\lib\index.js:508:7)
    at EventEmitter.emit (events.js:197:13)
    at EventEmitter.emit (domain.js:446:20)
    at DestroyableTransform.onError (C:\gulpdev\node_modules\gulp-cache\lib\index.js:347:15)
    at Object.onceWrapper (events.js:285:13)
    at DestroyableTransform.emit (events.js:197:13)
    at DestroyableTransform.EventEmitter.emit (domain.js:446:20)
    at C:\gulpdev\node_modules\through2-concurrent\through2-concurrent.js:41:14
    at imagemin.buffer.then.catch.error (C:\gulpdev\node_modules\gulp-imagemin\index.js:98:5)
```

**Comment by cpetta** [comment]
Removed gulp-cache from the imagemin pipeline.

```
.pipe(cache(imagemin([plugins])))
```
Changed to
```
.pipe((imagemin([plugins]))
```
As a result, started getting the following
```
'optamizeImages' errored after 386 ms
Error in plugin "gulp-imagemin"
Message:
    The system cannot find the path specified.

Details:
    errno: ENOENT
    code: ENOENT
    syscall: spawn C:\gulpdev\node_modules\jpegtran-bin\vendor\jpegtran.exe
    killed: false
    stdout:
    stderr: The system cannot find the path specified.

    failed: true
    signal: null
    cmd: C:\gulpdev\node_modules\jpegtran-bin\vendor\jpegtran.exe -copy none -optimize -outfile C:\Users\cpetta\AppData\Local\Temp\946f5cc2-5f83-4c14-9209-28d54f20a67a C:\Users\[my username]\AppData\Local\Temp\14e8a89a-6651-4532-bed9-86bc17f8644f
    timedOut: false
    fileName: C:\gulpdev\dev\images\main.jpg
    domainEmitter: [object Object]
    domain: [object Object]
    domainThrown: false
```

**Comment by cpetta** [comment]
I looked in ```node_modules/jpegtran-bin``` and noticed that the vender folder didn't exist.
I ran ```yarn add jpegtran-bin``` (```npm install jpegtran-bin```) which seems to have fixed the error.

**Comment by cpetta** [comment]
This thread exists because I wasn't able to find any information on the issue, and I wanted to document the steps I took to resolve the issue.
fixed for this project with 4992a15bd022b2a9b42f9fc50a5a4ec23a607c4f
